### DISCUSSION TITLE
Parte 2: Largo de ID

### DISCUSSION THREAD

**Discussion by martineyz** [IIC2133-PUC/2024-2#158] [Tarea 2] [answered]
Holaa, ¿los ID tienen rango de valores o largo definido? Veo que en los test todos tienen un largo parecido, pero no sé si es por como se crearon los test nomás o realmente tiñen un rango de valores los ID.

**Comment by 4n1t4** [comment, marked as answer]
Hola , no , sin embargo serán ints
